pom.xml

```diff
@@ -53,24 +53,24 @@
         <scylla.version />
         <java.version>8</java.version>
         <log4j.version>1.2.17</log4j.version>
-        <slf4j.version>1.7.25</slf4j.version>
-        <slf4j-log4j12.version>1.7.25</slf4j-log4j12.version>
+        <slf4j.version>1.7.36</slf4j.version>
+        <slf4j-log4j12.version>1.7.36</slf4j-log4j12.version>
         <guava.version>19.0</guava.version>
-        <netty.version>4.1.125.Final</netty.version>
+        <netty.version>4.1.127.Final</netty.version>
         <netty-tcnative.artifact>netty-tcnative-boringssl-static</netty-tcnative.artifact>
         <netty-tcnative.version>2.0.70.Final</netty-tcnative.version>
-        <metrics.version>3.2.2</metrics.version>
-        <snappy.version>1.1.10.5</snappy.version>
-        <lz4.version>1.4.1</lz4.version>
-        <hdr.version>2.1.10</hdr.version>
-        <jackson.version>2.15.2</jackson.version>
-        <joda.version>2.9.9</joda.version>
+        <metrics.version>3.2.6</metrics.version>
+        <snappy.version>1.1.10.7</snappy.version>
+        <lz4.version>1.8.0</lz4.version>
+        <hdr.version>2.2.2</hdr.version>
+        <jackson.version>2.15.4</jackson.version>
+        <joda.version>2.12.7</joda.version>
         <jsr353-api.version>1.0</jsr353-api.version>
         <jsr353-ri.version>1.0.4</jsr353-ri.version>
         <!-- Note:  When upgrading either jnr-ffi or jnr-posix, ensure that the versions are compatible.
              JNR has broken compatibility between minor versions in the past. -->
         <jnr-ffi.version>2.2.16</jnr-ffi.version>
-        <jnr-posix.version>3.1.15</jnr-posix.version>
+        <jnr-posix.version>3.1.19</jnr-posix.version>
         <groovy.version>2.4.21</groovy.version>
```

Comment:
I have not found any changelogs for this dep. I'm assuming the unit tests are sufficient check for this.